pages/getData.tsx

```diff
@@ -210,7 +244,9 @@ const GetData: NextPage = () => {
                 </div>
               </article>
               <pre style={{ wordBreak: 'break-all', whiteSpace: 'pre-wrap' }}>
-                {data}
+                {decodePermissionsData(data).map((element, index) => {
```

Comment:
this will try to decode the permissions for any data key.
You have to filter if the data key in the input is a data key related to permission or not.

See the screenshots below:

![image](https://user-images.githubusercontent.com/31145285/190430878-b83b4802-750b-4bcf-b134-0d00197a4b3d.png)

Here it decodes if it was permissions, but the data key is not related to permissions. ❌ 

![image](https://user-images.githubusercontent.com/31145285/190431604-7a12bdb0-139a-406e-bc7b-2d9d9107964a.png)